BaseProject.xcodeproj/project.pbxproj

```diff
@@ -814,7 +814,7 @@
 				INFOPLIST_FILE = BaseProject/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				OTHER_SWIFT_FLAGS = "-D PRODUCTION";
-				PRODUCT_BUNDLE_IDENTIFIER = com.baseproject.BaseProject;
+				PRODUCT_BUNDLE_IDENTIFIER = com.BaseProject.BaseProject;
```

Comment:
This bundle id is strange.

Reply:
In what sense do you consider this strange?

Reply:
Why would it be `com.BaseProject.BaseProject` and not `com.Wolox.BaseProject` ?

Reply:
Because this is the bundle id used for `production`, which is intended to be the build configuration used to deploy in client's account.

This way, we cannot use the same than we use for `release`, the build configuration used to deploy in wolox's account.

Given we cannot repeat bundle ids between apps, I thought it was a good way to distinguish them. Using the first part of the bundle id as organization name, and the second one as application name.

Reply:
Can't we swap `BaseProject` for `OrganizationName` then? Or does this affect other parts of the text/involves a major change?

Reply:
Yes, it involves a major change, since this way it gets defined during the renaming step in kickoff.

Reply:
Thought so 😅 let's keep it like this, since it's meant to be used with the script I don't see the need of commenting that unless we do it inside the guide.